.github/workflows/build.yml

```diff
@@ -73,7 +73,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.10.1]
+        python-version: ['3.10.9']
```

Comment:
I would suggest to put just '3.10' so if github decides to remove 3.10.9 it will continue working since specifying 3.10 will get the latest 3.10.

Reply:
Yes, I asked @maxipavlovic about that, he said that in version 3.10 there was a problem with parsing the only element in the list. So we are fixing the certain version.

Reply:
it fails if you do 3.10, but will work if you do '3.10' <= Funny eh? :)

Reply:
Wow! Ok, let's change it.
Thanks, Marc!

Reply:
Thanks @marcserrat !